src/peer-store/index.js

```diff
@@ -36,11 +36,15 @@ class PeerStore extends EventEmitter {
 
   /**
    * Stores the peerInfo of a new peer.
-   * If already exist, its info is updated.
+   * If already exist, its info is updated. If `silent` is set to
+   * true, no 'peer' event will be emitted. This can be useful if you
+   * are already in the process of dialing the peer. The peer is technically
+   * known, but may not have been added to the PeerStore yet.
    * @param {PeerInfo} peerInfo
+   * @param {boolean} [silent] (Default=false)
    * @return {PeerInfo}
    */
-  put (peerInfo) {
+  put (peerInfo, silent = false) {
```

Comment:
what do you think on transforming silent in an object? I think it helps to understand in the `peerStore.put(peerInfo, true)` what the `true` is

Reply:
Sounds good, I refactored and pushed a change to make it take an options object.